### ISSUE TITLE
Please add Virgin Media "Four" to the Ireland EPG

### ISSUE THREAD

**Issue by Loughlinstown** [iptv-org/epg#1520]
### Source

https://www.virginmediatelevision.ie/tv-guide

### Channels

-Four (aka "Virgin Media Four")

**Comment by Loughlinstown** [comment]
I have added this channel to the file but i have no experience with Github so have no idea how to create the pull request described in the contribution guide. Have attached the database zip here in case that helps?
[database-master.zip](https://github.com/iptv-org/epg/files/10253830/database-master.zip)

**Comment by PopeyeTheSai10r** [comment]
Hello @Loughlinstown, If you are unexperienced with Github and Pull Requests. You can read [Getting started with Guthub](https://docs.github.com/en/get-started) &  [About pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) to familiarize yourself.

Alternatively, you can an issue in the [database ](https://github.com/iptv-org/database)repository using the Add Channel template.

**Comment by niallobr** [comment]
@PopeyeTheSai10r is this channel available to grab using the epg-grabber package or is that not possible? I couldn't find it listed anywhere in the /sites folder. Apologies I know this is slightly unrelated to this specific issue.

**Comment by PopeyeTheSai10r** [comment]
Hey @niallobr, The station was added to the [Database](https://iptv-org.github.io/?q=VirginMediaFour.ie). We don't have a working source yet so can't be grabbed yet.

The other possible solutions is creating a config file for https://www.virginmediatelevision.ie/tv-guide or https://www.virginmediatv.ie/en/epg. As those are the only source I was able to find that have the station and it's information.
